doc/rtd/topics/faq.rst

```diff
@@ -135,12 +135,12 @@ that can validate your user data offline.
 
 .. _validate-yaml.py: https://github.com/canonical/cloud-init/blob/master/tools/validate-yaml.py
 
-Another option is to run the following on an instance when debugging:
+Another option is to run the following on an instance to debug userdata
+provided to the system:
 
```

Comment:
This is the doc update but it seemed an edit to an existing doc and didn't seem clear to me.

Reply:
Yeah, devel commands, and general CLI documentation, isn't great. In the devel case I had originally left it fairly undocumented as these `devel` commands will likely change form or break compatibility before we decide to make them fully fledged top-level (non-devel) subcommands of cloud-init.  But, now that we are referencing subcommand switches in docs in other places. it would be helpful to have a more full rendering of cli docs for this devel subcommands. I ultimately think we should autogeerate docs from the argparse configs so that we can keep that up to date any time the cli is changed.  I think `sphinx-argparse`  extension could do this for us, but I'd like to leave that to a separate PR as it'll potentially involve re-organizing the top-level [CLI docs](https://cloudinit.readthedocs.io/en/latest/topics/cli.html) docs. As I think we could try to modify how the CLI docs in general are exposed and documented.

Reply:
That said, I did just push a commit that renders cmdline devel schema to this branch. I think we can build on this if an acceptable format for all CLI commands.